build_tools/python/benchmark_suites/iree/mali_benchmarks.py

```diff
@@ -64,15 +65,18 @@ def generate(
     default_gen_configs = self._get_module_generation_configs(
         compile_config=self.DEFAULT_COMPILE_CONFIG,
         fp32_models=self.FP32_MODELS,
-        fp16_models=self.FP16_MODELS)
+        fp16_models=self.FP16_MODELS,
+        quant_models=self.QUANT_MODELS)
     fuse_padding_gen_configs = self._get_module_generation_configs(
         compile_config=self.FUSE_PADDING_COMPILE_CONFIG,
         fp32_models=self.FP32_MODELS,
-        fp16_models=self.FP16_MODELS)
+        fp16_models=self.FP16_MODELS,
+        quant_models=self.QUANT_MODELS)
```

Comment:
This one and the one below are compiled with the experimental fuse padding option, feel free to exclude these if you are not intended to also benchmark them.